trident.md

```diff
@@ -509,6 +509,15 @@ The specific semantics of the various ways to include or exclude entities are:
 
 If a query results in multiple individuals with the same name, forge will throw an error.
 
+##### Treatment of the genotype data while merging
+
+Forge performs a series of steps to merge the genotype data of multiple source files:
+
+1. Genotype data from each package is streamed in parallel and checks are performed that any SNP location present in any file is outputted (with missing data being filled in for packages where a particular SNP is missing), unless `--intersect` is given, in which case only those position which are present in every package are output.
+2. At each SNP, the consensus alleles are selected, by collecting all reference and alternative alleles from all sources. If more than two non-dummy alleles (alleles different from `N`) are present in that collection, an error is thrown. The two consensus alleles are then chosen and fixed to be reference and alternative allele.
```

Comment:
An error typically means the program stops or at least discards the element currently under consideration. Does the last sentence mean the program recovers from an error? Then it should be a warning, right?

Or should the last sentence be actually be the second sentence?

Reply:
Ok, I can clarify this further, but I am surprised this caused confusion. If more then two alleles are found, an error is thrown, which means the program stops immediately with an error message (we implement this via exceptions). Perhaps the last sentence should then start with an "Else" or so to make clear that "If only 2 or less non-N-alleles are listed, they are called "reference" and "alternative" and used for the output.